packages/contracts-bedrock/contracts/test/L1CrossDomainMessenger.t.sol

```diff
@@ -206,8 +205,24 @@ contract L1CrossDomainMessenger_Test is Messenger_Initializer {
         L1Messenger.xDomainMessageSender();
 
         address sender = Lib_PredeployAddresses.L2_CROSS_DOMAIN_MESSENGER;
-        uint256 senderSlotIndex = 1;
-        vm.store(address(op), bytes32(senderSlotIndex), bytes32(abi.encode(sender)));
+
+        uint256 senderSlotIndex = 50;
```

Comment:
I wish there was an easy way to prevent this from being brittle, sure its possible using `ffi` and grabbing the value from the storage layout snapshot but that is pretty slow. foundry has a `fs` library on the way, that could be used in the future

Reply:
I'm not worried, these values shouldn't change too much after this.